docs/howto/custom-management-commands.txt

```diff
@@ -256,6 +256,15 @@ All attributes can be set in your derived class and can be used in
     because attempting to set the locale needs access to settings. This
     condition will generate a :class:`CommandError`.
 
+.. attribute:: BaseCommand.style
+
+    An instance attribute that defines the style of the output, when writing to,
+    e.g., stdout.
```

Comment:
Maybe we should just say "stdout or stderr" (not sure there are any other places it's useful?)
I think some code examples as simple as:
`self.stdout.write(self.style.SUCCESS('...')`
would be great!